test/Dialect/HWArith/basic.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: circt-opt %s -verify-diagnostics | circt-opt -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL: @test1 {
+module @test1 {
```

Comment:
nit: I find it a bit strange seeing these ops inside a `module` (even though it is not illegal!). I think it would be more natural to place the ops inside their natural habitat, e.g. a `hw.module`, just to avoid any potential confusion from people just starting out in CIRCT.

Reply:
valid point